### PR TITLE
Add plugins: "Facts About Us", "Benefits", "How We Help", "Our Clients"

### PR DESCRIPTION
**Pull request by sendr**

### CODE REVIEW

File: core/models.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from cms.models.pluginmodel import CMSPlugin
+
+
+class Fact(CMSPlugin):
+    course_amount = models.CharField(max_length=5)
```
Comment (maxsocl) [inline core/models.py]: @sendr `course_amount` field should be Integer

screen shot 2017-03-01 at 1 18 47 pm

File: rg_promo/templates/plugin_templates/our_clients_item.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<div class="partners_item">
```
Comment (maxsocl) [inline rg_promo/templates/plugin_templates/our_clients_item.html]: @sendr there is a template bug there

screenshot at mar 01 13-30-39

File: rg_promo/templates/plugin_templates/facts_container.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{% load cms_tags %}
+<section class="facts">
+    <ul class="facts_list">
```
Comment (maxsocl) [inline rg_promo/templates/plugin_templates/facts_container.html]: @sendr Each fact has the same color

screenshot at mar 01 13-38-13

Comment (sendr) [inline rg_promo/templates/plugin_templates/facts_container.html]: done @maxsocl

File: core/models.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from cms.models.pluginmodel import CMSPlugin
```
Comment (sendr) [inline core/models.py]: done @maxsocl

File: core/cms_plugins.py [hunk 5]
```diff
@@ -0,0 +1,92 @@
+from cms.plugin_base import CMSPluginBase
+from cms.plugin_pool import plugin_pool
+from django.utils.translation import ugettext_lazy as _
+from core.models import Fact, Benefit, HowWeHelp, OurClient
```
Comment (sendr) [inline core/cms_plugins.py]: done localy @maxsocl

File: rg_promo/templates/plugin_templates/our_clients_item.html (hunk 2 above)
Comment (sendr) [inline rg_promo/templates/plugin_templates/our_clients_item.html]: done @maxsocl

File: core/cms_plugins.py [hunk 6]
```diff
@@ -1,7 +1,7 @@
 from cms.plugin_base import CMSPluginBase
 from cms.plugin_pool import plugin_pool
 from django.utils.translation import ugettext_lazy as _
-from core.models import Fact, Benefit, HowWeHelp, OurClient
+from . import models as core_models
```
Comment (sendr) [inline core/cms_plugins.py]: To not import each separate model

**Pull request merged**